dds/DCPS/RTPS/Spdp.cpp

```diff
@@ -2317,6 +2316,28 @@ Spdp::Interceptor::reactor_is_shut_down() const {
   return task_->is_shut_down();
 }
 
+void
+Spdp::schedule_send(const ACE_Time_Value& delay)
+{
+  class ScheduleCommand : public DCPS::ReactorInterceptor::Command {
+  public:
+    ScheduleCommand(Spdp* spdp, const ACE_Time_Value& delay) : spdp_(spdp), delay_(delay) {}
+
+  private:
+    void execute() {
+      if (-1 == spdp_->tport_->reactor()->schedule_timer(spdp_->tport_, this, delay_)) {
+        throw std::runtime_error("failed to schedule timer with reactor");
+      }
+    }
+
+    Spdp* const spdp_;
```

Comment:
`const Spdp* spdp_`?

Reply:
Moving `const` to the other side of `*` changes its meaning

Reply:
I forgot how to read pointers again. I was thinking like it was `Spdp const*` or `const Spdp*`. The next line should definitely be changed though.

Reply:
This is the baitclick of github review comments... I can't stand the suspense. Now I have to open the file to figure out what the next line is!

Reply:
clicked...and opened file to see next line.